VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimFileUpload/FileRequest.tsx

```diff
@@ -33,12 +33,21 @@ function FileRequest({ navigation, route }: FileRequestProps) {
   const theme = useTheme()
   const { t } = useTranslation(NAMESPACE.COMMON)
   const navigateTo = useRouteNavigation()
-  const { claimID } = route.params
-  const { data: claim, error: claimError, refetch: refetchClaim, isFetching: loadingClaim } = useClaim(claimID)
-  const requests = currentRequestsForVet(claim?.attributes.eventsTimeline || [])
+  const { claimID, claim } = route.params
```

Comment:
Seems like passing the claim as a prop is a reasonable solution. But shouldn't React Query be caching the claim to prevent this issue in the first place? If the user just visited the parent screen which fetched and cached the claim, why does fetching the same claim in the child component cause the loading indicator to appear? Caching should allow fetching the claim in the child component without having to pass it as a prop. Just trying to understand the root cause better

Reply:
it's because when this occurs is when the stale time is up, it isn't always here, but it does occur regularly enough that we should just pass the claim further down anyways